### PR TITLE
Update steal-systemjs to 0.19.1

### PR DESCRIPTION
**Pull request by m-mujica**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1274,19 +1274,16 @@ addStealExtension(function (loader) {
...
-if(typeof System !== "undefined") {
-	addEnv(System);
-}
-
-function addEnv(loader){
-	// Add the extension to _extensions so that it can be cloned.
-	loader._extensions.push(addEnv);
+// Steal Env Extension
+// adds some special environment functions to the loader
+addStealExtension(function (loader) {
```
Comment (matthewp) [inline main.js]: I think you maybe had a stale node_modules, this change isn't related to the steal-systemjs upgrade.

Comment (m-mujica) [inline main.js]: @matthewp I did a clean install and this comes up when I run grunt, maybe something we forgot to update before?

Comment (pYr0x) [inline main.js]: @matthewp i think you forgot to make a release after merging https://github.com/stealjs/steal/pull/1023 the last steal release is 7 days ago. the PR is 6 days ago.

Comment (matthewp) [inline main.js]: That change didn't need a release, it was just code reorganization.

Comment (pYr0x) [inline main.js]: yep, but i didnt run the grunt task because i thought you will run a `npm publish`

Comment (matthewp) [inline main.js]: Ok, no problem. I usually always run the grunt task when sending a PR. It's not an ideal system but I can't think of a better way right now.

Comment (pYr0x) [inline main.js]: ok good to know. i will right now always run the build and then make a PR

**Pull request merged**